config/webpack/test.js

```diff
@@ -1,13 +1,15 @@
 // config/webpack/test.js
+
 const environment = require('./environment')
-environment.plugins.get('Manifest').opts.writeToFileEmit = process.env.NODE_ENV !== 'test'
-environment.loaders.set('istanbul-instrumenter', {
-  test: /\.js$/,
-  enforce: "post",
-  loader: "istanbul-instrumenter-loader",
-  query: {
-    esModules: true
-  },
-  exclude: ["node_modules", /\.spec\.js$/]
-}) /* optional */
+// environment.plugins.get('Manifest').opts.writeToFileEmit = process.env.NODE_ENV !== 'test'
+// environment.loaders.set('istanbul-instrumenter', {
+//   test: /\.js$/,
+//   enforce: "post",
+//   loader: "istanbul-instrumenter-loader",
+//   query: {
+//     esModules: true
+//   },
+//   exclude: ["node_modules", /\.spec\.js$/]
+// }) // optional
+
```

Comment:
Ah, @cjcolvar was this the stuff that was causing the Circle CI build to break?   Previous karma config from webpacker docs that I put in and must have missed when cleaning up... if so.

Reply:
Yeah, my commit fixed the Circle CI build.  I used this issue as a hint: https://github.com/webpack/webpack/issues/4569.